src/main/java/com/ionic/sdk/agent/cipher/file/data/FileType.java

```diff
@@ -0,0 +1,52 @@
+package com.ionic.sdk.agent.cipher.file.data;
+
+/**
+ * Enumeration of all supported Ionic file cipher file types.
+ */
+public enum FileType {
+
+    /**
+     * Represents an unknown file type.
+     */
+    FILETYPE_UNKNOWN,
+
+    /**
```

Comment:
Does the documentation state what file types are supported?

Reply:
Do you have a suggestion?

Reply:
I just loaded the new javadoc. I see in the Javadoc in Enum FileType, that there is a list of Enum Constants.  It could be stated there which FileTypes are supported.

Reply:
Also, keep in mind that this source is already locked. We can't really make changes to the source in this pull request, as this isn't the repo we actually build from, and this ship has already sailed, so to speak.  However, any comments here can certainly be reflected in the upcoming 2.4 release (comments by Steven)